.travis.yml

```diff
@@ -16,8 +16,10 @@ before_script:
   - ansible --version
 
 env:
-  - SCENARIO=default
-  - SCENARIO=centos
+  - SCENARIO=default PSQL_VERSION=9.5
```

Comment:
Not sure if this is pretty, I bet someone who knows how travis-tox works can do this nicer. ;) But it seems to work.

Reply:
Looks good to me!